repairnator/pipeline-dockerimage4Bears/pipeline_launcher4Bears.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Use to create args in the command line for optionnal arguments
+function ca {
+  if [ -z "$2" ];
+  then
+      echo ""
+  else
+    if [ "$2" == "null" ];
+    then
+        echo ""
+    else
+        echo "$1 $2 "
+    fi
+  fi
+}
+
+args="`ca -g $GOOGLE_ACCESS_TOKEN``ca --spreadsheet $SPREADSHEET_ID``ca --dbhost $MONGODB_HOST``ca --dbname $MONGODB_NAME``ca --pushurl $PUSH_URL``ca --smtpServer $SMTP_SERVER``ca --notifyto $NOTIFY_TO`"
+
+if [ ! -d "$OUTPUT" ]; then
+    mkdir $OUTPUT
+fi
+
+# Clean env variables
+export GOOGLE_ACCESS_TOKEN=
+export SPREADSHEET_ID=
+export MONGODB_HOST=
+export MONGODB_NAME=
+export PUSH_URL=
+export SMTP_SERVER=
+export NOTIFY_TO=
+
+LOCAL_REPAIR_MODE=$REPAIR_MODE
```

Comment:
Actually for bears the repair mode will ALWAYS be "bears", right? So we don't need to specify it as a variable. And I can also remove it from the pipeline then. WDYT?

Reply:
I agree we don't actually need to specify it as a variable in the script since the scripts are separated now. However, I'm not sure if I understood on removing it from the pipeline. What do you mean?

Reply:
Could you please explain to me exactly what you thought?

Reply:
@surli here.

Reply:
> However, I'm not sure if I understood on removing it from the pipeline

I was just saying that we could remove the parameter from docker-pipeline script too. It should be hardcoded in the scripts of each docker-images: 
for docker-pipeline script:

`LOCAL_REPAIR_MODE=repair`

for docker-image4bears: 

`LOCAL_REPAIRMODE=bears`

Reply:
Please, see if my last commit does what you thought.